integration_test/gce/gce_testing.go

```diff
@@ -1001,7 +1001,11 @@ func addFrameworkMetadata(platform string, inputMetadata map[string]string) (map
 }
 
 func addFrameworkLabels(inputLabels map[string]string) (map[string]string, error) {
-	labelsCopy := make(map[string]string)
+	labelsCopy := map[string]string{
+		// Attach labels to automate cleanup
+		"env": "test",
```

Comment:
What's the significance of the "env" label? What's the use case for overriding it?